sale_automatic_workflow_advance/models/account_move.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import json
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def action_post(self):
```

Comment:
You can remove this, as the `sale_advance_payment` module will already implement it.